src/dispatch/signal/views.py

```diff
@@ -52,11 +53,14 @@ def get_signal_instances(common: CommonParameters):
 
 
 @router.post("/instances", response_model=SignalInstanceRead)
+@limiter.limit("250/minute")
```

Comment:
Why 250 and not a different value? Should we make it configurable via the Web UI?

Reply:
I chose this since it's only a bit higher to our limit set in `signal-process` without being too restrictive (it is more than double that limit, to be fair). I think it will take some tuning to get it just right, so open to ideas here.